components/ShopBuilder/SideBar.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+
+const Item = styled.div`
+  display: flex;
+  flex-direction: column;
+  font-size: 2rem;
+  text-align: center;
+  width: 100px;
+  height: 100px;
+  background: green;
+`;
+
+const SideBar = props => {
```

Comment:
I see a compeltely new side bar here. We might want to began styling it out more like our UXs designs.